IfcURIReference addition to IfcSimpleValue (#753)
buildingSMART/IFC4.3.x-development: docs/schemas/resource/IfcMeasureResource/Types/IfcSimpleValue.md

*  _IfcBoolean_: Defined type of simple type BOOLEAN.
*  _IfcLogical_: Defined type of simple type LOGICAL.
*  _IfcIdentifier_: Defined type of simple type STRING for identification purposes.
*  _IfcLabel_: Defined type of simple type STRING for naming purposes.
*  _IfcText_: Defined type of simple type STRING for descriptive purposes.
*  _IfcDateTime_: Defined type of simple type STRING to represent a date and time.
*  _IfcDate_: Defined type of simple type STRING to represent a date.
*  _IfcTime_: Defined type of simple type STRING to represent a time.
*  _IfcDuration_: Defined type of simple type STRING to represent a duration.
*  _IfcTimeStamp_: Defined type of simple type INTEGER to represent a point in time by seconds elapsed since 1970.

> *  _IfcURIReference_: Defined type of simple type STRING to represent a unique sequence of characters the identifies a logical or physical resource in the web technologies.

Reviewer: ```suggestion
*  _IfcURIReference_: Defined type of simple type STRING to represent a unique sequence of characters that identifies a logical or physical resource used by web technologies.
```

I see where you got the inspiration ;) https://en.wikipedia.org/wiki/Uniform_Resource_Identifier But I would copy it verbatim then. I personally have some doubts about the "unique sequence" (what does it mean, is it significant?) but can't argue with wikipedia.
